### PR TITLE
Update connection/invitation service to add pagination response

### PR DESCRIPTION
**Pull request by gongte1937**
Add pagination response for connection service and invitation service.

### CODE REVIEW

File: packages/sdk/test/services/connection-service.test.ts [hunk 1]
```diff
@@ -167,7 +167,7 @@ describe('ConnectionService', () => {
       })
       .add('connections', () => new ConnectionService(environment).list(testUserAuth))
       .it('decrypts connection name', ({ connections }) => {
-        for (const connection of connections) {
+        for (const connection of connections.connections) {
```
Comment (vijayshiyani) [inline packages/sdk/test/services/connection-service.test.ts:181]: @gongte1937 could you please check for the `meta` and `next_page_after` properties?

File: packages/sdk/src/services/invitation-service.ts [hunk 2]
```diff
@@ -27,9 +28,9 @@ export class InvitationService extends Service<InvitationApi> {
    * @param keypairId Use this public key in the new Connection. This is a Keystore Keypair.id (not external_id).
    * Throws an error if the key pair does not exist.
    */
-  public async list(credentials: IVaultToken & IKeystoreToken & IDEK & IKEK): Promise<{
-    invitations: Invitation[];
-  }> {
+  public async list(
+    credentials: IVaultToken & IKeystoreToken & IDEK & IKEK
+  ): Promise<InvitationsResponse> {
```
Comment (vijayshiyani) [inline packages/sdk/src/services/invitation-service.ts:33]: @gongte1937 could you please also update invitation list tests
https://github.com/Meeco/js-sdk/blob/develop/packages/sdk/test/services/invitation-service.test.ts#L41

**Review by vijayshiyani — CHANGES_REQUESTED** [review]

File: packages/sdk/test/services/invitation-service.test.ts [hunk 3]
```diff
@@ -38,14 +39,14 @@ describe('InvitationService', () => {
...
-          .reply(200, { invitations: [] });
+          .reply(200, { invitations: [], meta: {}, next_page_after: MOCK_NEXT_PAGE_AFTER });
     }
 
     customTest
       .mockCryppo()
       .nock('https://sandbox.meeco.me/vault', getInvitationsAPI())
       .do(() => new InvitationService(environment).list(testUserAuth))
-      .it('returns a list of invitations');
+      .it('returns a list of invitations',);
```
Comment (linasi) [inline packages/sdk/test/services/invitation-service.test.ts]: @gongte1937 is there a reason for this comma here?

File: packages/sdk/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@meeco/sdk",
   "description": "A wrapper library around Meeco's various API SDK's and encryption libraries",
-  "version": "4.20.0",
+  "version": "5.0.0",
```
Comment (linasi) [inline packages/sdk/package.json]: This change doesn't bring any major breaking changes. I suggest `4.21.0` as a new version number.

File: packages/sdk/test/services/invitation-service.test.ts (hunk 3 above)
Comment (gongte1937) [reply packages/sdk/test/services/invitation-service.test.ts]: @linasi  Yes, I was planing to create a function here but forgot to update, I have already updated the test here, now should be good.

File: packages/sdk/src/services/invitation-service.ts (hunk 2 above)
Comment (gongte1937) [reply packages/sdk/src/services/invitation-service.ts:33]: already updated

**Review by vijayshiyani — APPROVED** [review]

**Pull request merged**